addons/csi/kubevirt/csi-driver-operator.yaml

```diff
@@ -331,7 +331,7 @@ spec:
         runAsNonRoot: true
       containers:
         - name: manager
-          image: {{ Image "quay.io/kubermatic/kubevirt-csi-driver-operator:v0.3.0" }}
+          image: {{ Image "quay.io/kubevirt/csi-driver:latest" }}
```

Comment:
This was a custom operator before, now we just use the CSI driver? Is the upstream driver also an operator?

Reply:
Sorry, this was a mistake. Created new [PR](https://github.com/kubermatic/kubevirt-csi-driver-operator/pull/20/files) for updating the kubevirt-csi-driver daemonset